src/main/java/com/externalSorting/app/ExternalSortingRunner.java

```diff
@@ -35,21 +55,21 @@
         List<String> buffer = new ArrayList<>();
```

Comment:
Method `private static List<File> readAndSplitData(String filePath, File directoryPath)` doesn't have to be static. You create an object of this class and this object calls it's own method.